src/components/contexts/sync-handler.tsx

```diff
@@ -0,0 +1,61 @@
+import { Divider } from "@mui/material";
+import GenericDialog from "../generics/generic-dialog";
+import { DatePicker } from "@mui/x-date-pickers";
+import { DateTime } from "luxon";
+import strings from "../../localization/strings";
+
+/**
+ * Component properties
+ */
+interface Props {
+  syncStartDate: DateTime;
+  setSyncStartDate: (syncStartDate: DateTime) => void;
+  syncHandlerOpen: boolean;
+  setSyncHandlerOpen: (syncHandlerOpen: boolean) => void;
+  synchronizeTimeEntries: () => void;
+}
+
+/**
+ * SyncHandler component
+ *
+ * @param props component properties
+ */
+const SyncDialog = ({
+  syncStartDate,
+  setSyncStartDate,
+  syncHandlerOpen,
+  setSyncHandlerOpen,
+  synchronizeTimeEntries
+}: Props) => {
+  /**
+   * Component render
+   */
+  return (
+    <GenericDialog
+      open={syncHandlerOpen}
+      onClose={() => {
+        setSyncHandlerOpen(false);
+      }}
+      onCancel={() => {
+        setSyncHandlerOpen(false);
+      }}
+      onConfirm={() => {
+        synchronizeTimeEntries();
+        setSyncHandlerOpen(false);
+      }}
+      cancelButtonText={strings.syncDialog.cancel}
+      confirmButtonText={strings.syncDialog.sync}
+      title={strings.syncDialog.title}
+    >
+      <DatePicker
+        label={strings.syncDialog.label}
+        maxDate={DateTime.now()}
```

Comment:
We should also have a minDate, in the old time bank this was 31/07/2021 so lets have the same set here

Reply:
Added minDate to here as well. Did you mean 31/08/2021? Because in the old timebank-ui the date object is declared as `new Date(2021, 7, 31)`.